events/calendars/views.py

```diff
@@ -0,0 +1,23 @@
+from django.shortcuts import render
+from django.views.generic.base import View
+from django.http import JsonResponse
+import json
+from event.models import User, EventUserAssignment, Event
+
+# Create your views here.
+class Calendar(View):
+    """
+    View used .
+    """
+    def get(self, request, user_id=None):
+        if not user_id:
```

Comment:
You need to get user from request. Please do not pass user_id as param. Modify url pattern as well